src/connecpy/wsgi.py

```diff
@@ -235,26 +236,14 @@ def _get_endpoint(self, path_info):
 
         return endpoint
 
-    def handle_error(self, exc, environ, start_response):
+    def handle_error(self, exc, _environ, start_response):
         """Handle and log errors with detailed information."""
-        if isinstance(exc, exceptions.ConnecpyServerException):
-            status = {
-                errors.Errors.InvalidArgument: "400 Bad Request",
```

Comment:
Previously only these three errors had correct HTTP status. Now they all should

Reply:
I see 👍